ci/docker/concourse-runner/Dockerfile

```diff
@@ -10,26 +10,19 @@ RUN apk add --no-cache \
   python3 \
   ca-certificates \
   jq \
-  # dockerd dependencies
-  iptables \
-  util-linux 
-
-RUN apk add --no-cache \
   openjdk11 \
   npm \
   maven \
   tini \
-  --repository=http://dl-cdn.alpinelinux.org/alpine/edge/community
-
-RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
-RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.31-r0/glibc-2.31-r0.apk
-RUN apk add glibc-2.31-r0.apk
```

Comment:
🔥

Reply:
How do we know all these could be deleted?